HTML5 build error with Emscripten 1.39.5: 'timestamp' undefined, should be replaced by emscripten_get_now()

**Godot version:** Git 318c69351624f7794c51b5385d252af397c0404a

**OS/device including version:** Fedora 30, Emscripten 1.39.5

**Issue description:** I get a build error when compiling for HTML5:

```text
Compiling ==> platform/javascript/os_javascript.cpp
platform/javascript/os_javascript.cpp:315:28: error: no member named 'timestamp' in 'EmscriptenMouseEvent'
                uint64_t diff = p_event->timestamp - os->last_click_ms;
                                ~~~~~~~  ^
1 error generated.
shared:ERROR: '/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/bin/clang++ -target wasm32-unknown-emscripten -D__EMSCRIPTEN_major__=1 -D__EMSCRIPTEN_minor__=39 -D__EMSCRIPTEN_tiny__=5 -D_LIBCPP_ABI_VERSION=2 -Dunix -D__unix -D__unix__ -Werror=implicit-function-declaration -Xclang -nostdsysteminc -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/include/libcxx -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/lib/libcxxabi/include -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/include/compat -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/include -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/include/libc -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/lib/libc/musl/arch/emscripten -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/local/include -Xclang -isystem/root/.emscripten_cache/wasm-obj/include -c -std=c++11 -Os -fno-exceptions -fno-rtti -Wall -Werror=return-type -DJAVASCRIPT_ENABLED -DUNIX_ENABLED -DNO_THREADS -DNO_SAFE_CAST -DJAVASCRIPT_EVAL_ENABLED -DNDEBUG -DPTRCALL_ENABLED -DGDSCRIPT_ENABLED -DMINIZIP_ENABLED -DZSTD_STATIC_LINKING_ONLY -DFREETYPE_ENABLED -DSVG_ENABLED -DHAVE_MNTENT -Ithirdparty/nanosvg -Ithirdparty/freetype/include -Ithirdparty/libpng -Ithirdparty/zstd -Ithirdparty/zlib -Iplatform/javascript -I. -DEMSCRIPTEN platform/javascript/os_javascript.cpp -fno-inline-functions -Xclang -isystem/builds/Calinou/godot-builds-ci/godot/emsdk/upstream/emscripten/system/include/SDL -c -o platform/javascript/os_javascript.javascript.opt.bc -mllvm -combiner-global-alias-analysis=false -mllvm -enable-emscripten-sjlj -mllvm -disable-lsr' failed (1)
scons: *** [platform/javascript/os_javascript.javascript.opt.bc] Error 1
scons: building terminated because of errors.
```

[Full build log on GitLab CI](https://gitlab.com/Calinou/godot-builds-ci/-/jobs/389971570)

**Steps to reproduce:** Build Godot with `scons platform=javascript`.

--- a/platform/javascript/os_javascript.cpp
+++ b/platform/javascript/os_javascript.cpp
@@ -301,6 +301,7 @@ EM_BOOL OS_JavaScript::mouse_button_callback(int p_event_type, const EmscriptenM
 	ev->set_position(correct_canvas_position(p_event->canvasX, p_event->canvasY));
 	ev->set_global_position(ev->get_position());
 	dom2godot_mod(p_event, ev);
+
 	switch (p_event->button) {
 		case DOM_BUTTON_LEFT: ev->set_button_index(BUTTON_LEFT); break;
 		case DOM_BUTTON_MIDDLE: ev->set_button_index(BUTTON_MIDDLE); break;
@@ -312,7 +313,7 @@ EM_BOOL OS_JavaScript::mouse_button_callback(int p_event_type, const EmscriptenM
 
 	if (ev->is_pressed()) {
 
-		uint64_t diff = p_event->timestamp - os->last_click_ms;
+		double diff = emscripten_get_now() - os->last_click_ms;
 
 		if (ev->get_button_index() == os->last_click_button_index) {
 

--- a/platform/javascript/os_javascript.h
+++ b/platform/javascript/os_javascript.h
@@ -55,7 +55,7 @@ class OS_JavaScript : public OS_Unix {
 	Point2 touches[32];
 
 	Point2i last_click_pos;
-	uint64_t last_click_ms;
+	double last_click_ms;
 	int last_click_button_index;
 
 	MainLoop *main_loop;

